src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -74,7 +74,8 @@ function getRuleContextsFromTrackedFilters({
         // An empty object is technically not a `SearchResults` but `getRefinements`
         // only accesses properties, meaning it will not throw with an empty object.
         helper.lastResults || ({} as SearchResults),
-        sharedHelperState
+        sharedHelperState,
+        true
```

Comment:
`getRefinements` is not added in this PR, but what if we rename the third parameter to something like `includeQuery`?
https://github.com/algolia/instantsearch.js/blob/feat/rule-filters-query/src/lib/utils/getRefinements.ts#L96:L96

So far there were only two cases calling `getRefinements` to see which refinements to "clear". I guess that's how it ended up having that parameter name.

Of course, not a blocker of the PR.

Reply:
yes, the parameter is called like that because of the old "clearsQuery" option for current refinements, but it's indeed confusing now that we don't use it anymore. Let's change it!